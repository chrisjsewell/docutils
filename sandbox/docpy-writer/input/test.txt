General text markup for rst to docpy
====================================

admonitions
'''''''''''

reST supports: attention, caution, danger, error, hint, important,
note, tip, warning.

docpy has a notice environment in two variants, note and warning.

.. note:: admonitions should become a ``note notice``

.. warning:: danger is everywhere.

.. error:: other admonitions will/might not work properly.

