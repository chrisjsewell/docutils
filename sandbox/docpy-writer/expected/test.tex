\ifx\locallinewidth\undefined\newlength{\locallinewidth}\fi
\setlength{\locallinewidth}{\linewidth}


\section{General text markup for rst to docpy\label{-general-text-markup-for-rst-to-docpy}}

reST supports: attention, caution, danger, error, hint, important,
note, tip, warning.

docpy has a notice environment in two variants, note and warning.
\begin{notice}[note]
admonitions should become a \code{note notice}
\end{notice}
\begin{notice}[warning]
danger is everywhere.
}}
\end{sffamily}
\end{center}
\begin{center}\begin{sffamily}
\fbox{\parbox{\admonitionwidth}{
\textbf{\large Error}
\vspace{2mm}

other admonitions will/might not work properly.
}}
\end{sffamily}
\end{center}

